crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -136,19 +172,36 @@ impl SequencerBlockData {
     /// # Errors
     ///
     /// - if the block has no data hash
+    /// - if the block has no transactions
```

Comment:
This list is great when eventually moving to typed errors, which we should really do...